.changeset/popular-roses-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+'@makeswift/runtime': patch
+---
+
+Add RichText control for usage within custom components 
```

Comment:
This feels rather simple for a release as important as this one, don't you think?

Reply:
Updated it with more details.